git_theta/git_utils.py

```diff
@@ -26,58 +30,13 @@ def get_git_repo():
     return git.Repo(os.getcwd(), search_parent_directories=True)
 
 
-def get_git_theta(repo, create=False):
-    """
-    If create argument is true, create $git_root/.git_theta and return path
-    Otherwise return $git_root/.git_theta path
-
-    Parameters
-    ----------
-    git_root : str
-        path to git repository's root directory
-    create : bool
-        argument to create the directory
-
-    Returns
-    -------
-    str
-        path to $git_root/.git_theta directory
-    """
-    git_theta = os.path.join(repo.working_dir, ".git_theta")
-    if not os.path.exists(git_theta) and create:
-        logging.debug(f"Creating git theta directory {git_theta}")
-        os.makedirs(git_theta)
-    return git_theta
-
-
-def get_git_theta_model_dir(repo, model_path, create=False):
-    """
-    If create is true, create directory under $git_root/.git_theta/ to store a model and return path
-    Otherwise just return path that stores a model
-
-    Parameters
-    ----------
-    repo : git.Repo
-        Repo object for the current git repository
-
-    model_path : str
-        path to model file being saved
-
-    create : bool
-        argument to create the directory
-    Returns
-    -------
-    str
-        path to $git_root/.git_theta/$model_path directory
-    """
-    git_theta = get_git_theta(repo)
-    git_theta_model_dir = os.path.join(git_theta, model_path)
-
-    if not os.path.exists(git_theta_model_dir) and create:
-        logging.debug(f"Creating model directory {git_theta_model_dir}")
-        os.makedirs(git_theta_model_dir)
-
-    return git_theta_model_dir
+def set_hooks():
+    repo = get_git_repo()
+    hooks_dst = os.path.join(repo.git_dir, "hooks")
+    package = importlib.resources.files("git_theta")
+    for hook in ["pre-push", "post-commit"]:
+        with importlib.resources.as_file(package.joinpath("hooks", hook)) as file:
+            shutil.copy(file, hooks_dst)
```

Comment:
Yeah this implementation will just replace any existing hooks (either a user's hooks or the git-theta hooks put there by a previous command) any time a git-theta command (except install) is run. I'll add a check in to only copy if the hook scripts are not already there. I think for the second problem (overwriting existing user hooks), we should take an approach similar to this https://stackoverflow.com/questions/26624368/handle-multiple-pre-commit-hooks but since this is a bit more involved let's make this it's own issue.